system-variables.md

```diff
@@ -610,9 +610,16 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 ### `tidb_enable_index_merge` <span class="version-mark">从 v4.0 版本开始引入</span>
 
 - 作用域：SESSION | GLOBAL
-- 默认值：`OFF`
+- 默认值：`ON` 
 - 这个变量用于控制是否开启 index merge 功能。
 
+> **注意：**
+>
+> 1. 版本号小于 v4.0.0 的集群升级到版本号大于等于 v5.4.0 的集群，开关默认保持关闭。防止升级后计划发生变化导致回退。
+> 2. 版本号大于等于 v4.0.0 的集群升级到版本号大于等于 v5.4.0 的集群，开关保持升级前的状态。
+> 3. 版本号大于等于 v5.4.0 的新建集群，开关默认保持开启。
+
+
```

Comment:
```suggestion
> - 当集群从 v4.0.0 以下版本升级到 v5.4.0 及以上版本时，该变量开关默认关闭，防止升级后计划发生变化导致回退。
> - 当集群从 v4.0.0 及以上版本升级到 v5.4.0 及以上版本时，该变量开关保持升级前的状态。
> - 对于 v5.4.0 及以上版本的新建集群，该变量开关默认开启。

```
The modification is as follows:
1. Optimized description about the version information
2. Changed the ordered list into an unordered list.
3. Deleted a redundant blank line.